### PR TITLE
Update changelog for PR #54

### PR DESCRIPTION
**Pull request by nmeirik**
As pr request in https://github.com/backup-manager/symfony/pull/54#issuecomment-449623389

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.1.4
```
Comment (Nyholm) [inline Changelog.md]: Must be 2.2.0

Comment (nmeirik) [reply Changelog.md]: Fixed

**Comment by Nyholm** [comment]
No, I had two things. You only fixed one of them. 

I want you to go to "released" and click on the "3 commits" to see what changes since the last version. 

screenshot 2018-12-24 at 09 28 56

That will get you to this page: https://github.com/backup-manager/symfony/compare/2.1.3...master

As you can see there has been 3 commits since last release. We dont need to mention that the readme has been updated. But we do need to say that we added support for B2. (That is a new feature and that is why we have to do a minor update instead of a patch update as you first suggested)

**Comment by nmeirik** [comment]
Ah, I see. Thanks for taking the time to explain. I've added the B2 support to the changelog.

**Comment by Nyholm** [comment]
Excellent. Thank you. 

I will tag a new version now but note that it will not be updated automatically because backup-manager/backup-manager needs to be releasing 1.3 first. 

You can of course do: 

```
# Composer.json
"backup-manager/backup-manager": "dev-master as 1.3.0",
```

**Comment by nmeirik** [comment]
For reference: https://github.com/backup-manager/backup-manager/issues/148

**Pull request merged**